CONTENT/program.markdown

```diff
@@ -3,9 +3,9 @@
 [](#english)
 # Wizardry & Magic Spells
 
-## All aspects of magic interesting to a fresh wizard.
+## Everything you need to know about our Magic sessions
 
-We are going to try hard to make this learning journey exciting for all future wizard amigos and provide the following learning materials open source and under a free license, to help you master the magic spells to become an active problem solver:
+We are trying hard to make this learning journey exciting for all future wizard amigos. All the learning materials are open source and under a free license. We cover the main elements that every wizard apprentice needs to know:
```

Comment:
I like it - but I remember my yoda saying or my sister... do or do not, there is no "trying" :-) So maybe that should be changed too.

I would like to see the `active problem solver` piece in there. I like that `line 8` was seperated into more smaller sentences. I dont like `main elements`, maybe it can be phrased to avoid that - this is what i tried in the former version.